include/swift/AST/DiagnosticsSema.def

```diff
@@ -6699,5 +6699,31 @@ ERROR(macro_undefined,PointsToFirstBadToken,
       "macro '%0' is undefined; use `-load-plugin-library` to specify dynamic "
       "libraries that contain this macro", (StringRef))
 
+//------------------------------------------------------------------------------
+// MARK: Move Only Errors
+//------------------------------------------------------------------------------
+
+ERROR(moveonly_copyable_type_with_moveonly_subtype, none,
+      "copyable %0 %1 can not contain a move only subtype. Found move only "
+      "subtype in %2 '%3.%4'",
+      (DescriptiveDeclKind, DeclName, DescriptiveDeclKind, StringRef, StringRef))
+ERROR(moveonly_copyable_type_with_moveonly_subtype2, none,
+      "copyable %0 %1 can not contain a move only type",
+      (DescriptiveDeclKind, DeclName))
+ERROR(add_nominal_moveonly_conformance,none,
+     "consider applying the '@_moveOnly' attribute %0 %1",
+     (DescriptiveDeclKind, DeclName))
```

Comment:
This should be a note, and explain the effect of applying the attribute. It's also not really a conformance. How about:

"apply the '@_moveOnly' attribute to make %1 move-only"